src/reynir_correct/tools/diffchecker.py

```diff
@@ -44,15 +44,15 @@ def main():
     # Hægt að biðja um annað til að fá frekari upplýsingar!
     options["format"] = "text"  # text, json, csv, m2
     options["annotations"] = True
-
     options["all_errors"] = True
-
     # options["infile"] = open("prufa.txt", "r")
-    options["one_sent"] = True
+    options["one_sent"] = False
     # options["generate_suggestion_list"] = True
-    options["ignore_comments"] = False
-    # options["annotate_unparsed_sentences"] = False
-
+    options["ignore_comments"] = True
```

Comment:
What's this?

Reply:
An example of an input file: 
# Test á efni úr Ritmyndum
Þú vilt aðskilað á kosnað aðahalds.

This option only exists here, is used to not print the comments in the output.